### ISSUE TITLE
Footer / Add accessibility statement page

### ISSUE THREAD

**Issue by ainoc** [CSCfi/swift-browser-ui#874]
Add a page about accessibility statements (fi and en versions) and add link to this page to the footer.

**Comment by csc-felipe** [comment]
@ainoc, What should be in this page?

**Comment by csc-felipe** [comment]
On hold pending material and validating the issue.

**Comment by ainoc** [comment]
Check that statement is up to date.
